src/client/css/style.scss

```diff
@@ -0,0 +1,88 @@
+$btnColor: #4A90E2;
+$formColor: #FFFFFF;
+$bgColor: rgb(246,249,255);
+$defaultColor: #9B9B9B;
+$titleFont: Helvetica-Bold;
+
+* {
+  margin: 0;
+}
+
+body {
+  background: $bgColor;
+}
+
+header{
+  width: 100%;
+  height: 57.2px;
+  background: $btnColor;
+
+  button {
+    margin: 13.55px 0 13.55px 13.55px;
```

Comment:
use only integers for pixels. And also try to avoid the pixel settings. Use em, rem, % anything, but this will broken on smaller screens